h5c/vic/src/vic-webapp/e2e/vch-create-wizard/2-multi-dvswitch.e2e-spec.ts

```diff
@@ -35,10 +35,10 @@ import {
 } from './common';
 
 describe('VCH Create Wizard with multiple DV Switches', () => {
-  jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_TIMEOUT;
+  jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_TIMEOUT * 4;
```

Comment:
Why not just update the value for JASMINE_TIMEOUT in the constants file? Aren't we using the extended timeout for both test sets now? That or add another one that is defined as an extended version.

Reply:
JASMINE_TIMEOUT is set to 30 seconds and is used in many unit test cases. I don't feel like updating the value to 2 minutes because that a unnecessarily long timeout for unit tests might risk masking the real reason for a test failure. Timeout of 2 minutes is only enforced for the Protractor test cases right now.

Reply:
Then maybe we need one like PROTRACTOR_TIMEOUT. Idk this just seems a bit sloppy to me but I guess it's minor nit.

Reply:
`PROTRACTOR_TIMEOUT` sounds like a good alternative.